src/compositor/meta-shaped-texture.c

```diff
@@ -906,6 +911,15 @@ meta_shaped_texture_set_fallback_size (MetaShapedTexture *self,
   priv->fallback_height = fallback_height;
 }
 
+void
+meta_shaped_texture_set_cull_back_face (MetaShapedTexture *self, gboolean cull)
+{
+  MetaShapedTexturePrivate *priv = self->priv;
```

Comment:
You probably want to add:

    g_return_if_fail (META_IS_SHAPED_TEXTURE (self));

here.